samples/django.md

```diff
@@ -32,6 +32,8 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
    FROM python:3
    ENV PYTHONUNBUFFERED=1
    WORKDIR /code
+   RUN apk update \
+    && apk add postgresql-dev gcc python3-dev musl-dev
```

Comment:
This doesn't look correct. The example here uses `python:3` as a base image, which is an alias for `python:3-buster`, and based on a Debian base image, not alpine; https://github.com/docker-library/python/blob/d18aeb0a2f1e52d8ff82dfe57104943174a0d8ab/3.9/buster/Dockerfile

<img width="301" alt="Screenshot 2021-08-06 at 12 04 05" src="https://user-images.githubusercontent.com/1804568/128494425-d8fcf23a-6fd7-49b5-8de9-b80412a54289.png">

I suspect you were trying to optimise the example and create a smaller image by using alpine, but the current Dockerfile looks correct